### DISCUSSION TITLE
GitLab / Autoclose dashboard issue

### DISCUSSION THREAD

**Discussion by jokay** [renovatebot/renovate#18154] [Request Help] [answered]
Shouldn't the env param `RENOVATE_DEPENDENCY_DASHBOARD` set to **false** autoclose all existing Dependency Dashboard issues of all repositories on GitLab?

**Comment by rarkins** [comment, marked as answer]
The source is here: https://github.com/renovatebot/renovate/blob/2de27aeddac7d6b395248491dcbf0364f3c3b829/lib/workers/repository/dependency-dashboard.ts#L178-L200

You should make sure that all of the conditions in the `if` return `false`.

**Reply by jokay** [reply]
I manually closed them *(and hope they were not recreated)* ^^
